charters/DataProcessingService/DataProcessingService.md

```diff
@@ -0,0 +1,51 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+The data processing service is a service that executes automated transformation to scientific data through cloud-based execution of scientific pipelines. It can be used in any instantiation of the Data Coordination Platform (DCP). 
```

Comment:
"Transformative" is emphasized here. Maybe its meaning should be defined. What does this imply about this service? 

Should this be:
```diff
+The data processing service is a service that executes automated transformation of scientific data through cloud-based execution of scientific pipelines
-The data processing service is a service that executes automated transformation to scientific data through cloud-based execution of scientific pipelines
```

Reply:
+1 for to to of

Reply:
added a definition and reworded.